domains/darcoz.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "a website",
+    "repo": "https://github.com/DarcoZav/Darcoz.github.io",
+    "owner": {
+        "username": "DarcoZav",
+        "email": "zarconxorp@gmail.com",
+        "twitter": "ZarcoZxacy"
+    },
+    "record": {
+        "CNAME": "darcozav.github.io"
```

Comment:
404?

Reply:
Sorry I forgot to remove the 404.html, and now it's fixed

Reply:
@DarcoZav It still returns 404.

Reply:
Wait I make new repo

Reply:
I already remove the 404

Reply:
@DarcoZav I still get a 404 when trying to reach https://darcozav.github.io

Reply:
Live site? 
https://darcozav.github.io/Darcoz.github.io/

Reply:
@DarcoZav Rename the repo to `darcozav.github.io`

Reply:
@WilliamDavidHarrison done, i really sorry about that